### PR TITLE
Balder/no more vds configuration

### PR DESCRIPTION
**Pull request by baldersheim**
@vekterli PR

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/ContentBuilderTest.java [hunk 1]
```diff
@@ -206,7 +202,7 @@ public void handleSingleNonSearchPersistentVds() throws Exception {
         assertFalse(s.hasIndexedCluster());
         assertTrue(s.getClusters().isEmpty());
 
-        assertTrue(a.getPersistence() instanceof VDSEngine.Factory);
+        assertTrue(a.getPersistence() instanceof ProtonEngine.Factory);
```
Comment (vekterli) [inline config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/ContentBuilderTest.java]: Test name is `handleSingleNonSearchPersistentVds` (emphasis "Vds"); should we remove it entirely or just rename it?

**Review by vekterli — APPROVED** [review]
:+1:

**Comment by vekterli** [comment]
Merging to get factory's thoughts on the matter, although all VDS system tests should have been rewritten or removed by now.

**Comment by baldersheim** [comment]
I will rename unless we have another test that covers the no-search case, or it is redundant in some other way.

**Pull request merged**